Directory.Build.props

```diff
@@ -43,7 +43,7 @@
     <!-- Suppress the "Microsoft.Bcl.Memory 9.0.0 doesn't support net6.0/net7.0 and has not been tested with it" warning -->
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
   </PropertyGroup>
-
+  
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == 'NET6_0_OR_GREATER'">
```

Comment:
Consider updating these as well.

Reply:
This looks incorrect.

TargetFrameworkIdentifier should be compared against .NETFramework, .NETCoreApp, or .NETStandard

Reply:
IsTargetFrameworkCompatible would be a better fit here agreed.

Reply:
The property group should be removed, it isn't doing anything, and IsAotCompatible is the recommended way to handle this.

Reply:
Same with the next block containing EnableMicrosoftExtensionsConfigurationBinderSourceGenerator. The condition is wrong, it will never be used. This is contributing to why some of these assemblies are not AOT compatible.